code/modules/projectiles/projectile/energy.dm

```diff
@@ -93,6 +92,58 @@
 	tesla_zap(src, zap_range, power, zap_flags)
 	qdel(src)
 
+/obj/item/projectile/energy/bsg
+	name = "orb of pure bluespace energy"
+	icon_state = "bluespace"
+	impact_effect_type = /obj/effect/temp_visual/bsg_kaboom
+	damage = 60
+	damage_type = BURN
+	range = 9
+	weaken = 1 //This is going to knock you off your feet
+	eyeblur = 5
+	speed = 2
+	alwayslog = TRUE
+
+/obj/item/ammo_casing/energy/bsg/ready_proj(atom/target, mob/living/user, quiet, zone_override = "")
+	..()
+	var/obj/item/projectile/energy/bsg/P = BB
+	spawn(1)
+		P.chain = P.Beam(user, icon_state = "sm_arc_supercharged", icon = 'icons/effects/beam.dmi', time = 1000, maxdistance = 30)
```

Comment:
```suggestion
		P.chain = P.Beam(user, icon_state = "sm_arc_supercharged", icon = 'icons/effects/beam.dmi', time = 100 SECONDS, maxdistance = 30)
```

(100 seconds seems a bit long though)

Reply:
will make it 10 seconds, not that it will last that long due to range

Reply:
will make it 10 seconds, however projectile will expire before then anyway